frontend/packages/dev-console/src/utils/pipeline-augment.ts

```diff
@@ -110,7 +111,7 @@ export const getResources = (data: PropPipelineData[]): Resource => {
       if (pipeline.metadata && pipeline.metadata.namespace && pipeline.metadata.name) {
         propsReferenceForRuns.push(`PipelineRun_${i}`);
         resources.push({
-          kind: 'PipelineRun',
+          kind: referenceForModel(PipelineRunModel),
```

Comment:
This was correct before

Reply:
Getting error ```No model registered for PipelineRun```. If I just pass ```'PipelineRun'````

Reply:
Oops, yeah this is for `Firehose`. Your change is right here. My fault.

We should rename that prop honestly. It's not a `kind`.